app/bundles/CoreBundle/Assets/css/app/less/components/alerts.less

```diff
@@ -14,6 +14,69 @@
   }
 }
 
+.alert {
+  border-width: 0 0 0 3px;
+  color: var(--text-primary);
+  display: flex;
+  flex-wrap: nowrap;
+  gap: 3px;
+  background-color: var(--layer);
+  border-radius: var(--border-radius-sm);
+
+  &.alert-success,
+  &.alert-info,
+  &.alert-warning,
+  &.alert-danger {
+    &:not([class*="col-"]):before {
+      font-family: 'remixicon';
+      display: inline-flex;
+      position: relative;
+      font-size: 22px;
+      padding: 0px 3px;
+      margin-right: 10px;
+      line-height: .5em;
+      top: .15em;
+    }
+  }
+
+  &.alert-success { 
```

Comment:
You should avoid nesting classes where possible. Makes it easier to read if there's less levels, but also easier to override properties whenever needed.
Especially since the class names are already variant names, they should be able to stand alone.
```
.alert {
// stuff
}

.alert-success,
.alert-blablabla {
  // stuff
}

.alert-success {
  // stuff
}

Reply:
Nice idea, thanks!